code/modules/clothing/shoes/magboots.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/clothing/shoes/magboots
 	name = "magboots"
-	desc = "Magnetic boots, often used during extravehicular activity to ensure the user remains safely attached to the vehicle."
+	desc = "Magnetic boots with a heavy pull, often used during extravehicular activity to ensure the user remains safely attached to the vehicle."
```

Comment:
is this... really necessary? Wouldn't players come to this conclusion that magnetic boots pull really hard just by using them in the first place?

Reply:
Fair point, I was just transferring that little snippet from the functionally identical syndicate magboots.